src/Template/Layout/js/app/components/dialog/dialog.js

```diff
@@ -43,40 +43,83 @@ export const BeditaDialog = {
         };
     },
     methods: {
-        show(message, headerText = this.dialogType) {
+        show(message, headerText = this.dialogType, root = document.body) {
             this.headerText = headerText;
             this.message = message;
             this.isOpen = true;
+            if (!this.$el) {
+                this.$mount();
+            }
+            if (this.$el.parentNode !== root) {
+                root.appendChild(this.$el);
+            }
         },
-        hide() {
+        hide(destroy = false) {
             this.dialogType = '';
             this.isOpen = false;
+
+            if (this.destroyOnHide || destroy) {
```

Comment:
Isn't it better to invert this check? If you pass `false` from outside, checking `this.destroyOnHide` (that defaults to `true`), the dialog will be still destroyed